libs/wingsdk/src/util/util.ts

```diff
@@ -183,9 +184,8 @@ export class Util {
     opts?: ExecOptions
   ): Promise<Output> {
     const execOpts = {
-      cwd: opts?.cwd,
       windowsHide: true,
-      shell: false,
```

Comment:
Enabling shell has security implications, so we should avoid it for `exec()`.

I think `shell` should be `false` for `exec()` and `true` for `shell()`.

Reply:
You cannot set `shell: true` for `shell()` since it's a `string` option.

![image](https://github.com/winglang/wing/assets/10464497/b4faf64b-5a09-4255-88d1-13caecc46706)

But we can keep the explicit `shell: false` for `exec()` if you prefer despite the fact that it's the default setting.